build.sbt

```diff
@@ -68,24 +54,24 @@ lazy val playPublishingSettings: Seq[sbt.Setting[_]] = Seq(
   Compile / packageSrc / publishArtifact  := false
 )
 
-def oneForkedJvmPerTest(tests: Seq[TestDefinition]): Seq[Group] =
-  tests map { test =>
-    Group(
-      test.name,
-      Seq(test),
-      SubProcess(
-        ForkOptions().withRunJVMOptions(Vector(s"-Dtest.name=${test.name}"))
-      )
-    )
-  }
-
 Global / bloopAggregateSourceDependencies := true
 
-commands ++= Seq(
-  Command.command("run-all-tests") { state => "test" :: "it:test" :: state },
-
-  Command.command("clean-and-test") { state => "clean" :: "compile" :: "run-all-tests" :: state },
+lazy val it = (project in file("it"))
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(
+    name := "integration-tests",
+    Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-eT"),
+    inConfig(Test)(BloopDefaults.configSettings),
```

Comment:
we don't need this as bloop is already set up for Test (I believe this works for all subprojects.....)